src/include/kernel/abi.h

```diff
@@ -28,8 +28,8 @@
 #define __KERNEL_ABI_H__
 
 /** \brief SOF ABI version major, minor and patch numbers */
-#define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 13
+#define SOF_ABI_MAJOR 4
```

Comment:
This is a big change.

Reply:
Yea, but I'm not sure about it.

From one side, old kernel version with new firmware (with extended manifest) won't work - bacause of additional data at the begin of firmware binary -> error during loading new firmware - so from this point it should be major change. But introduced changes will fall firmware load before ABI compatibility check, so changing major ABI number doesn't help in such situation.

From other side, when new kernel (with extended manifest routines) recognize that given firmware is in old version without extended manifest (by checking magic number at begin of firmware binary), then it it can be easily handled and be backward compatible, so from this point it shouldn't be major change